version/CURRENT_VERSION

```diff
@@ -1 +1 @@
-1.0.0-beta.6
+1.0.0-beta
```

Comment:
This change will affect the versioning as follows:
- alpha/beta/etc releases made automatically use whatever is defined here together with a timestamp suffix
- releases made by hand won't require bumping the build for beta/rc/etc releases made by hand
- regular releases will have the full version in this file (1.0.0, 1.1.0, 1.0.2)

This makes it a lot easier to release beta builds.